plugins/linode/personal_access_token.go

```diff
@@ -0,0 +1,39 @@
+package linode
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func PersonalAccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.PersonalAccessToken,
+		DocsURL:       sdk.URL("https://www.linode.com/docs/products/tools/cloud-manager/guides/cloud-api-keys/"),
+		ManagementURL: sdk.URL("https://cloud.linode.com/profile/tokens"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "Token used to authenticate to Linode.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 64,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+		Importer: importer.TryAll(
+			importer.TryEnvVarPair(defaultEnvVarMapping),
```

Comment:
Would be nice in the future to also support importing from [`~/.config/linode-cli`](https://www.linode.com/docs/products/tools/cli/guides/install/#interactive-configuration), but we can also do that in a follow-up PR. Created an [issue](https://github.com/1Password/shell-plugins/issues/91) for it.